go/libraries/doltcore/sqle/logictest/regressions.sql

```diff
@@ -0,0 +1,158 @@
+CREATE TABLE `mysql_results` (
```

Comment:
File level comment: I don't really care about using mysql results as a baseline here. But maybe I'm wrong. Why do we care about MySQL results?